src/main/target/common_pre.h

```diff
@@ -184,6 +184,7 @@
 #define USE_MSP_DISPLAYPORT
 #define USE_MSP_OVER_TELEMETRY
 #define USE_LED_STRIP
+#define USE_STATS
```

Comment:
This is overflowing the flash space on some F3 targets - we have to move this to a feature cut level for F3 where this does not happen any more.

Reply:
The failing target is on level 1 already... 
Will enable since F4 then.